src/main/java/com/trello/LogInPage.java

```diff
@@ -44,6 +47,15 @@ public void logInWithEmail(String userEmail) {
         waitSeconds(3);
     }
 
+    public void openLogInPage() {
+        openLogInPageButton.click();
+    }
+
+    public String getLoginButtonLocation() {
+        return logInButton.getLocation()
```

Comment:
.getLocation() returns **Point** object, we can't do .toString() for it.
But you can do .getX() and .getY() from Point in separate methods.